From 634ddd73f790dd779e97edd1491178897858f7c0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Thu, 10 Jul 2014 22:34:43 -0400
Subject: [PATCH] man: document x-systemd.device-timeout for crypttab

https://bugs.freedesktop.org/show_bug.cgi?id=54210
(cherry picked from commit dc5cd2b77288dc85c82cb95810cd14caee311e7e)
---
 man/crypttab.xml | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/man/crypttab.xml b/man/crypttab.xml
index 9dbf1546b1..d658a6ff20 100644
--- a/man/crypttab.xml
+++ b/man/crypttab.xml
@@ -319,6 +319,21 @@
                         </varlistentry>
 
                         <varlistentry>
+                                <term><option>x-systemd.device-timeout=</option></term>
+
+                                <listitem><para>Specifies how long
+                                systemd should wait for a device to
+                                show up before giving up on the
+                                entry. The argument is a time in
+                                seconds or explicitly specifified
+                                units of <literal>s</literal>,
+                                <literal>min</literal>,
+                                <literal>h</literal>,
+                                <literal>ms</literal>.
+                                </para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
                                 <term><option>tmp</option></term>
 
                                 <listitem><para>The encrypted block device will
